src/tasks/runstart.ts

```diff
@@ -925,6 +926,32 @@ export const RunStartQuest: Quest = {
       },
       combat: new CombatStrategy().macro(Macro.trySkill($skill`Blow the Purple Candle!`).default()),
     },
+    {
+      name: "Bakery Pledge",
+      ready: () => have($familiar`Patriotic Eagle`) && haveFreeRunSource(),
+      completed: () =>
+        have($effect`Citizen of a Zone`) ||
```

Comment:
We should update `ITEMS.md` to reflect that we are going to be supporting this with the eagle